man/fi_efa.7.md

```diff
@@ -194,6 +194,9 @@ These OFI runtime parameters apply only to the RDM endpoint.
 *FI_EFA_SET_CUDA_SYNC_MEMOPS*
 : Set CU_POINTER_ATTRIBUTE_SYNC_MEMOPS for cuda ptr. (Default: 1)
 
+*FI_EFA_INTER_MIN_READ_MESSAGE_SIZE*
+: The minimum message size for inter EFA read message protocol. If instance support RDMA read, messages whose size is larger than this value will be sent by read message protocol. (Default 1048576).
```

Comment:
nit: Add unit to the size similar to
```
FI_EFA_RUNT_SIZE : The maximum number of bytes that will be eagerly sent by inflight messages uses runting read message protocol (Default 307200).
```
i.e.
```
The minimum message size in bytes for inter EFA read message protocol. If instance support RDMA read, messages whose size is larger than this value will be sent by read message protocol. (Default 1048576).
```

Reply:
done